examples/python3.yaml

```diff
@@ -0,0 +1,27 @@
+# configuration document version
+version: "1"
+
+# Configuration shared by all bots.
+# The fields can be overriden in the bot configuration - see below
+global:
+  notifications:
+    email_addresses: [phracek@redhat.com]
+
+upstream-to-downstream:
+  # is betka enabled for this repository
+  # optional - defaults to true
+  enabled: true
+
+  notifications:
+    email_addresses: [rh-software-collections@redhat.com]
```

Comment:
internal ML, probably should not keep the address in here. Change to sclorg@redhat.com or remove entirely. Same for the other files.

Reply:
Remove them, is better